chrome/content/zotero/xpcom/attachments.js

```diff
@@ -539,39 +541,28 @@ Zotero.Attachments = new function(){
 		}
 		
 		// Save using a hidden browser
-		var nativeHandlerImport = function () {
-			return new Zotero.Promise(function (resolve, reject) {
-				var browser = Zotero.HTTP.loadDocuments(
-					url,
-					Zotero.Promise.coroutine(function* () {
-						try {
-							let attachmentItem = yield Zotero.Attachments.importFromDocument({
-								libraryID,
-								document: browser.contentDocument,
-								parentItemID,
-								title,
-								collections,
-								saveOptions
-							});
-							resolve(attachmentItem);
-						}
-						catch (e) {
-							Zotero.logError(e);
-							reject(e);
-						}
-						finally {
-							Zotero.Browser.deleteHiddenBrowser(browser);
-						}
-					}),
-					undefined,
-					(e) => {
-						reject(e);
-					},
-					true,
+		var nativeHandlerImport = async function () {
+			try {
+				let browser = await HiddenBrowser.create(url, {
+					requireSuccessfulStatus: true,
+					docShell: { allowImages: true },
```

Comment:
I didn't test, but is this actually doing anything if we're just getting and reparsing the document HTML? Seems like it might matter if we needed to run SingleFile in the actor, but not as is.